### PR TITLE
Make expiration time a module parameter

### LINKED ISSUE #1081: Make expiration time a module parameter
Now that we have timeout for lambda execution, I think it makes sense to force everything have the same exp. This also allows us to figure out which reporter fails to report data in the future. Let's have 20 blocks as the default value.

Update everything that includes expiration time to not have it.

### PR DESCRIPTION
**Pull request by Benzbeeb**
Fixed #1081

### CODE REVIEW

File: chain/db/request.go [hunk 1]
```diff
@@ -4,6 +4,7 @@ import (
 	"strconv"
 
 	"github.com/bandprotocol/bandchain/chain/x/oracle"
+	"github.com/bandprotocol/bandchain/chain/x/oracle/types"
```
Comment (taobun) [inline chain/db/request.go]: Remove this line

File: chain/db/request.go [hunk 2]
```diff
@@ -169,12 +170,16 @@ func (b *BandDB) handleMsgRequestData(
 	if err != nil {
 		return err
 	}
+	request, err := b.OracleKeeper.GetRequest(b.ctx, types.RequestID(id))
```
Comment (taobun) [inline chain/db/request.go]: Use `oracle.RequestID`

Comment (taobun) [reply chain/db/request.go]: but we have 2 place to add new request (ibc.go), so in my opinion change this on `AddNewRequest` function instead

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by sorawit — DISMISSED** [review]
Thank you!

**Review by sorawit — APPROVED** [review]

**Pull request merged**